python/pfs/drp/stella/calibrateWavelengthsTask.py

```diff
@@ -202,23 +236,24 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.debugInfo = lsstDebug.Info(__name__)
 
-    def fitWavelengthSolution(self, spec, detectorMap, rng=np.random):
-        """Fit wavelength solution for a spectrum
-
+    def fitWavelengthSolution(self, spec, detectorMap,rng=np.random):
+    
+        """
+        Fit wavelength solution for a spectrum
+        
         Parameters
         ----------
-        spec : `pfs.drp.stella.Spectrum`
-            Spectrum to fit; updated with solution.
+        spectrumSet : `pfs.drp.stella.SpectrumSet`
+            SpectrumSet to fit; updated with solution.
```

Comment:
In general, you should avoid making changes like this that aren't necessary. If you think they are necessary, they should go in a separate commit.